### PR TITLE
replace zh to /zh in content/zh/docs/tutorials/ directory

### PR DESCRIPTION
**Pull request by tanjunchen**
replace zh to /zh in content/zh/docs/tutorials/ directory

### CODE REVIEW

**Comment by tanjunchen** [comment]
/cc @tengqm 

done

File: content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md [hunk 1]
```diff
@@ -453,10 +453,10 @@ Do not leave your WordPress installation on this page. If another user finds it,
 
 {{% capture whatsnext %}}
 
-* Learn more about [Introspection and Debugging](zh/docs/tasks/debug-application-cluster/debug-application-introspection/)
-* Learn more about [Jobs](zh/docs/concepts/workloads/controllers/jobs-run-to-completion/)
-* Learn more about [Port Forwarding](zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
-* Learn how to [Get a Shell to a Container](zh/docs/tasks/debug-application-cluster/get-shell-running-container/)
+* Learn more about [Introspection and Debugging](/zh/docs/tasks/debug-application-cluster/debug-application-introspection/)
+* Learn more about [Jobs](/zh/docs/concepts/workloads/controllers/jobs-run-to-completion/)
```
Comment (tengqm) [inline content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md:457]: no. .... these should not be changed.

Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md:457]: why?

Comment (tengqm) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md:457]: they are the English original text, why change them?

Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md:457]: First of all, adding prefixes in the zh directory without much impact.
Secondly, it is convenient to replace.
The situation to consider for url replacement is complicated, and I have to reconsider.
I have to reconsider it. 
I remember that the Chinese link of istio.io introduced the ci mechanism.
/hold
I need to check it carefully

File: content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md [hunk 2]
```diff
@@ -453,24 +453,16 @@ Do not leave your WordPress installation on this page. If another user finds it,
 
 {{% capture whatsnext %}}
 
-* Learn more about [Introspection and Debugging](zh/docs/tasks/debug-application-cluster/debug-application-introspection/)
-* Learn more about [Jobs](zh/docs/concepts/workloads/controllers/jobs-run-to-completion/)
-* Learn more about [Port Forwarding](zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
-* Learn how to [Get a Shell to a Container](zh/docs/tasks/debug-application-cluster/get-shell-running-container/)
+<!--
```
Comment (tanjunchen) [inline content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md:456]: attention here

File: content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: The reason I am suggesting leaving the English content untouched is for future. When the upstream English content is changed again, someone will step up and do the comparison, i.e. what has changed and what hasn't. If we change the English content in the comment, comparison for difference would become a nightmare. Agreed?

Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: see https://github.com/k8smeetup/website-tasks/issues/3699 

Here is a direct comparison of the differences between different branches of the original English, but adding the prefix /zh directly to the English in the comment is also not a good way to handle it.
It's a bit tricky to handle.

Comment (tengqm) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: So ... please consider change back all the links modified in the commented out English originals. It causes more problem than provide any convenience.

File: content/zh/docs/tutorials/_index.md [hunk 3]
```diff
@@ -57,10 +57,10 @@ Before walking through each tutorial, you may want to bookmark the
 * [Introduction to Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
 -->
 
-* [你好 Minikube](zh/docs/tutorials/hello-minikube/)
+* [你好 Minikube](/zh/docs/tutorials/hello-minikube/)
 
 <!--
-* [Hello Minikube](zh/docs/tutorials/hello-minikube/)
+* [Hello Minikube](/zh/docs/tutorials/hello-minikube/)
```
Comment (tengqm) [inline content/zh/docs/tutorials/_index.md:63]: 我关心的问题是：第60行的改动是正确的，但是我们为什么要改动63行呢？英文原文的 63 行是没有 'zh/' 或者 '/zh/' 前缀的。这样改动之后，下次再做比较，在英文 master 和中文 master 之间寻找英文更新内容的时候，会发现很多不同步的问题。

File: content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md (hunk 1 above)
Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: sure .

File: content/zh/docs/tutorials/_index.md (hunk 3 above)
Comment (tanjunchen) [reply content/zh/docs/tutorials/_index.md:63]: 好的，好在目前只修改了 /zh/docs/tutorials/ 目录 ，我会尽快修复这个问题的。

File: content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md (hunk 2 above)
Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md:456]: 注意下这里原文中文汉化存在问题

File: content/zh/docs/tutorials/_index.md [hunk 4]
```diff
@@ -16,17 +16,17 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-Kubernetes 文档的这一部分包含教程。一个教程展示了如何完成一个比单个[任务](zh/docs/tasks/)更大的目标。
+Kubernetes 文档的这一部分包含教程。一个教程展示了如何完成一个比单个[任务](/zh/docs/tasks/)更大的目标。
```
Comment (tanjunchen) [inline content/zh/docs/tutorials/_index.md:19]: 这个pr的问题是在   content/zh/docs/tutorials/ 目录下， 汉化链接前补 / ，英文原始注解下 去掉以前添加的 zh 或者 /zh

**Comment by tengqm** [comment]
Many thanks for this clean up.
/lgtm
/approve

**Pull request merged**